### ISSUE TITLE
FutureWarning: this method is deprecated in favour of `Styler.to_html()`

### ISSUE THREAD

**Issue by Arian96669** [dexplo/dataframe_image#37]
With recent changes to Pandas (I'm using version 1.4.0), Styler.render() is being depreciated in favour of Styler.to_html().

Inside _pandas_accessor.py, change:
`68    if is_styler:`
`69        html = '<div>' + obj.render() + '</div>'`

to
` 68    if is_styler: `
`69        html = '<div>' + obj.to_html() + '</div>'`

That got rid of the future warning for me.

**Comment by fredericomattos** [comment]
> Thank you. Will update when I have time.

Hi @tdpetrou same here mate:

```
lib\site-packages\dataframe_image\_pandas_accessor.py:69: FutureWarning: this method is deprecated in favour of `Styler.to_html()`
  html = '<div>' + obj.render() + '</div>'
```

**Comment by moshemoshe137** [comment]
Fixed by 7204ece
